libs/core/src/components/sage-button/test/sage-button.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { SageButton } from '../sage-button';
+
+describe('sage-button', () => {
+  it('renders', async () => {
+    const page = await newSpecPage({
+      components: [SageButton],
+      html: `<sage-button></sage-button>`,
+    });
+    expect(page.root).toEqualHtml(`
+      <sage-button>
+        <mock:shadow-root>
+          <slot></slot>
+        </mock:shadow-root>
+      </sage-button>
```

Comment:
Don't be afraid to fix the failing test. It's fairly simple. The answer is in the output 😉